src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -43,6 +44,7 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
   const [showStageGateModal, setShowStageGateModal] = useState<boolean>(false);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const dropdownOpen = Boolean(anchorEl);
+  const { isLoading } = useCheckDescriptionBullet();
```

Comment:
Instead of adding this react hook here, I think you should do something similar to my other comment in the checklist component where it calls useCheckDescriptionBullet() on line 38. Then when it disables the checkbox, add an or for the isLoading value there.